src/main/webapp/IssuePrescription.jsp

```diff
@@ -0,0 +1,44 @@
+<%-- 
+    Document   : IssuePrescription
+    Created on : 15-Dec-2020, 17:17:29
+    Author     : Bredan
+--%>
+
+<%@page contentType="text/html" pageEncoding="UTF-8"%>
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+        <title>Issue Prescription Form</title>
+    </head>
+    <body>
+        <h1>Issue Prescription</h1>
+        <form method="post" action="${pageContext.request.contextPath}/IssuePrescription">
+            <label>
+                Patient ID: <input name="submitted-patientid">
```

Comment:
As per my other comment, look at the appointment and approvals pages to see how to do a combo-box with the User names in it for the user, which submits User IDs when submitted.

Reply:
You should also probably add a `required` to these.

Reply:
required tags added in!